Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4262.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 4262, "Label HorizontalTextAlignment=\"Center\" not working in conjunction with LineHeight on iOS", PlatformAffected.iOS)]
+	public class Issue4262 : ContentPage
+	{
+		public Issue4262()
+		{
+			var label = new Label() { Text = "This is center aligned&#x0a;line 2.", HorizontalTextAlignment = TextAlignment.Center };
+			var label2 = new Label() { Text = "This is center aligned&#x0a;line 2.", HorizontalTextAlignment = TextAlignment.Center, LineHeight = 1.5 };
```

Comment:
```suggestion
			var label2 = new Label() { Text = "If this is not center aligned, this test has failed.", HorizontalTextAlignment = TextAlignment.Center, LineHeight = 1.5 };
```

Reply:
Make the test case more explicit for anyone who has to manually run it (since we can't easily automate this test).

Reply:
done